tests/drivers/uart/uart_mix_fifo_poll/prj.conf

```diff
@@ -1,11 +1,17 @@
 CONFIG_SERIAL=y
 CONFIG_ZTEST=y
-CONFIG_TEST_USERSPACE=y
+CONFIG_TEST_USERSPACE=n
 CONFIG_ZTEST_THREAD_PRIORITY=5
 CONFIG_MAIN_STACK_SIZE=2048
 CONFIG_ZTEST_STACK_SIZE=2048
 CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_COUNTER=y
 
-# CONFIG_NO_OPTIMIZATIONS=y
+CONFIG_OUTPUT_DISASSEMBLY=y
+#CONFIG_NO_OPTIMIZATIONS=y
```

Comment:
I think some of configs here can be removed. Additionally, do we need disassembly output here?

Reply:
This PR becomes an superset of all changes (uart + tests). I will be opening clean PR with only driver changes (PM) since that is what we need now. Tests will come later with less pressure.